doctr/models/backbones/mobilenet/tensorflow.py

```diff
@@ -244,32 +205,12 @@ def mobilenetv3_large(pretrained: bool = False) -> MobileNetV3_Large:
         >>> out = model(input_tensor)
 
     Args:
+        small: if True, instantiate small architecture (default), else large architecture
         pretrained: boolean, True if model is pretrained
 
     Returns:
         A  mobilenetv3_large model
     """
-
-    return _mobilenetv3_large('mobilenetv3_large', pretrained)
-
-
-def mobilenetv3_small(pretrained: bool = False) -> MobileNetV3_Small:
-    """MobileNetV3 architecture as described in
-    `"Searching for MobileNetV3",
-    <https://arxiv.org/pdf/1905.02244.pdf>`_.
-
-    Example::
-        >>> import tensorflow as tf
-        >>> from doctr.models import mobilenetv3_small
-        >>> model = mobilenetv3_small(pretrained=False)
-        >>> input_tensor = tf.random.uniform(shape=[1, 512, 512, 3], maxval=1, dtype=tf.float32)
-        >>> out = model(input_tensor)
-
-    Args:
-        pretrained: boolean, True if model is pretrained
-
-    Returns:
-        A  mobilenetv3_small model
-    """
-
-    return _mobilenetv3_small('mobilenetv3_small', pretrained)
+    if small:
+        return _mobilenetv3('mobilenetv3_small', pretrained)
+    return _mobilenetv3('mobilenetv3_large', pretrained)
```

Comment:
Better to create two different functions `mobilenet_v3_small` and `mobilenet_v3_large` that will call `_mobilenetv3` with the correct string